MetaMask/metamask-extension
# fix: Add Basic Functionality to Settings Search

## **Description**

Adds "Basic Functionality" to the Settings search

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/25185?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/25441

## **Manual testing steps**

1. Go to Settings
2. Search the term "Basic"
3. See "Basic Functionality" and click it
4. End up on top of the Privacy tab

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
